src/main/java/duke/program/Duke.java

```diff
@@ -52,20 +54,46 @@ private void exit() {
 
     /**
      * Duke's behaviour loop after set-up to before exit.
-     * @return void
      */
     private void loop() {
         boolean isExiting = false;
         Command c;
```

Comment:
While straightforward here, using single char variable names can become bad habit in future. Try using a more verbose variable name to avoid confusion later on.